domain/src/main/java/edu/nextstep/camp/calculator/domain/util/CalculatorInputUtil.kt

```diff
@@ -0,0 +1,14 @@
+package edu.nextstep.camp.calculator.domain
```

Comment:
```suggestion
package edu.nextstep.camp.calculator.domain.util
```
패키지명을 명확하게 적어주시면 좋을 것 같아요 (이하 관련 내용 동일)